### PR TITLE
FIX #2652 Splash Screen Stretched

### LINKED ISSUE #2652: Splash Screen Stretched
**Describe the bug**
 Splash screen is displayed when a user opens the app, on small screen devices the splash screen is working as expected, but on large screen devices, the splash screen is getting stretched and doesn't work as expected.

**To Reproduce**
1. Go to the Oppia app
2. Click on the icon
3. If your device screen is more than 6 inch
4. See  splash screen getting stretched

**Expected behavior**
 The splash screen should be displayed in the same aspect ratio as needed.

**Screenshots**

[Note: Also reported in #3193 -- see additional videos in that issue.]

Expected Behaviour
Expected Behaviour
Actual Behaviour
SplashScreen

**Device**
- OnePlus 7 Pro
- Android 10.0

The logo should be fixed as per device width:
- Mobile ([Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/ddc5589e-10d1-48e4-b551-5487801233e1), [Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/375d3300-f70d-44ed-a48a-269a58ecc854))
- Tablet ([Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d39fdc8a-2bee-4867-a1a6-ce4280671c2b), [Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d69bb715-1aea-4815-8401-6e7fde167ed4))

### PR DESCRIPTION
**Pull request by backpackerdeveloper**
FIX #2652 Splash Screen Stretched 
**Changes**:
 - Added SDP library in dependency
 - in SplashScreenActivity : in ImageView Changed the dp to sdp
 - Now it fits on different sizes of devices
 
 **ABOUT SDP**
An android lib that provides a new size unit - sdp (scalable dp). This size unit scales with the screen size. It can help with supporting multiple screens.

### CODE REVIEW

**Comment by backpackerdeveloper** [comment]
Signed the CLA sheet 👍

**Comment by adhiamboperes** [comment]
Hey @backpackerdeveloper, can you tell me about any alternative solutions you considered before adding the sdp lib?

Please also read the wiki on [submitting a PR.](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR) You deleted the PR template but you need to fill it out instead. Please look at [this](https://github.com/oppia/oppia-android/pull/5148) other PRs in the repo and copy the PR description template and add it to this PR.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Hey @backpackerdeveloper, can you tell me about any alternative solutions you considered before adding the sdp lib?

Please also read the wiki on [submitting a PR.](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR) You deleted the PR template but you need to fill it out instead. Please look at https://github.com/oppia/oppia-android/pull/5148 other PRs in the repo and copy the PR description template and add it to this PR.

**Comment by backpackerdeveloper** [comment]
@adhiamboperes As I saw ConstraintLayout is properly, assigned, so I directly sdp lib, and it worked.
and yes I was slightly confused with the PR Template at that time, and mistakenly deleted the template

File: gradle/wrapper/gradle-wrapper.jar
Comment (adhiamboperes) [inline gradle/wrapper/gradle-wrapper.jar:1]: This should not be part of the commit.

File: gradle/wrapper/gradle-wrapper.properties
Comment (adhiamboperes) [inline gradle/wrapper/gradle-wrapper.properties:1]: Please remove this change, we do not want to update gradle at the moment.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Hi @backpackerdeveloper, thanks for this PR. There are a number of things that need work in order for us to accept this solution:

- [ ] The library needs to be added to bazel as well as gradle. Please follow [this wiki](https://github.com/oppia/oppia-android/wiki/Updating-Maven-Dependencies) to add a dependency to bazel.
- [ ] We do not need to update gradle now, so please exclude any gradle wrapper update changes from your PR.
- [ ] Please refer to the [wiki on submitting a PR](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR). Your branch is called develop, and that is prohibited in the repo.
- [ ] Add before and after screenshots to your PR, to demostrate the changes.(Also make sure you read the essential checklist because it has important requirements)

This means that you should close this PR and open a new one with an appropriately named branch, and also please address all these comments in the new PR.

**Pull request closed without merging**